pset-functions.js

```diff
@@ -31,6 +31,25 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(pScore, dScore) {
+   let result
+
+   if ((pScore>21)&&(dScore>21)) {
+      result = 0
+   } else if (pScore>21) {
+      result = dScore
+   } else if (dScore>21) {
+      result = pScore
+   } else if (dScore >= pScore) {
+      result = dScore
+   } else {
+      result = pScore
+   }
+   return result
+}
+
+console.log(blackJack(21, 21))
```

Comment:
Lance, great use of if/else statements to catch all possible combinations of outcomes here! It is possible to return within the if/else blocks to give you what you want in each scenario; this will allow you to eliminate the need to declare a variable. For an example, take a look [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L33)